kafka/testhelpers.go

```diff
@@ -33,15 +34,19 @@ import (
 import "C"
 
 var testconf struct {
-	Docker       bool
-	Semaphore    bool
-	Brokers      string
-	Topic        string
-	GroupID      string
-	PerfMsgCount int
-	PerfMsgSize  int
-	Config       []string
-	conf         ConfigMap
+	Docker        bool
+	Semaphore     bool
+	Brokers       string
+	BrokersSasl   string
+	SaslUsername  string
+	SaslPassword  string
+	SaslMechanism string
+	Topic         string
```

Comment:
This will break existing test configuration files that everyone is using
It's not a part of any API guarantees or anything, though, I suppose it's mostly internal... so should I make this change regardless of that?

Reply:
If it is internal only, then we should make the change.